SALib/sample/ff.py

```diff
@@ -76,30 +76,24 @@ def generate_contrast(problem):
 
 def sample(problem):
     """Generates model inputs using a fractional factorial sample
-
```

Comment:
These blank lines should be retained in the docstring to comply with numpydoc style.